libs/build_api_v2.py

```diff
@@ -37,7 +37,7 @@ def _build_actuals(actual_data: dict) -> Actuals:
         hospitalBeds={
             "capacity": actual_data.get(CommonFields.MAX_BED_COUNT),
             "currentUsageCovid": actual_data.get(CommonFields.CURRENT_HOSPITALIZED),
-            "currentUsageTotal": actual_data.get(CommonFields.CURRENT_HOSPITALIZED_TOTAL),
+            "currentUsageTotal": actual_data.get(CommonFields.HOSPITAL_BEDS_IN_USE_ANY),
```

Comment:
I opened https://trello.com/c/cyWTrrDs/753-clean-up-commonfields-currenthospitalizedtotal-vs-hospitalbedsinuseany-confusion to clean this up fully, but as-is the currentUsageTotal field in the API is always null because combined data sets only pulls in HOSPITAL_BEDS_IN_USE_ANY.

I noticed this when I plumbed in the HHS data for counties and it didn't show up in the API how I expected.